assets/prototype/domain/eventEditor/containers/mutations/datetimes/useDateimeQueryOptions.js

```diff
@@ -0,0 +1,21 @@
+import useEventId from '../../queries/useEventId';
+import { queries } from '../../queries';
+
+const { GET_DATETIMES } = queries;
+
+const useDateimeQueryOptions = () => {
```

Comment:
This is a well-thought improvement to avoid duplications and verboseness  :medal_sports: